src/vs/workbench/parts/markers/electron-browser/markersTreeViewer.ts

```diff
@@ -249,6 +249,11 @@ export class MarkerRenderer implements ITreeRenderer<Marker, MarkerFilterData, I
 		dom.toggleClass(templateData.code.element, 'marker-code', !!marker.code);
 		templateData.code.set(marker.code || '', codeMatches);
 
+		if (marker.url) {
+			templateData.code.element.parentElement.setAttribute('href', marker.url);
+			templateData.code.element.parentElement.setAttribute('target', '_blank');
+		}
```

Comment:
Here, the link is on the brackets as well, compared to the two other views. With the highlight color on top, this doesn't look good. Suggestions welcome!